extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -58,27 +67,53 @@ export class SubscriptionService {
   initialize_() {
     const pageConfigResolver = new PageConfigResolver(this.ampdoc_.win);
 
-    // TODO(@prateekbh): read this config from the document.
-    const platformConfigs = [
-      {
-        paywallUrl: '/subscription/1/entitlements',
-      },
-      {
-        paywallUrl: '/subscription/2/entitlements',
-      },
-    ];
-
-    return pageConfigResolver.resolveConfig().then(pageConfig => {
+    const configPromises = Promise.all([
+      this.getServiceConfig_(),
+      pageConfigResolver.resolveConfig(),
+    ]);
+
+    return configPromises.then(promiseValues => {
+      const serviceConfig = user().assert(promiseValues[0]);
+      this.serviceConfig_ = serviceConfig;
+      const pageConfig = user().assert(promiseValues[1]);
       this.pageConfig_ = pageConfig;
-      platformConfigs.forEach(platformConfig => {
-        this.subscriptionPlatforms_.push(
-            new LocalSubscriptionPlatform(
-                this.ampdoc_,
-                platformConfig,
-                pageConfig
-            )
-        );
+
+      dev().assert(this.serviceConfig_['services'],
+          'Services not configured in service config');
+
+      this.serviceConfig_['services'].forEach(service => {
+        this.initializeSubscriptionPlatforms_(service, pageConfig);
+      });
+    });
+  }
+
+  /**
+   * @param {!JsonObject} serviceConfig
+   * @param {!PageConfig} pageConfig
+   * @private
+   */
+  initializeSubscriptionPlatforms_(serviceConfig, pageConfig) {
+    if (serviceConfig['authorizationUrl']) {
```

Comment:
This should be that `if ((serviceConfig['serviceId'] || 'local') == 'local') {...}`